js/modules/authDOMHandler.js

```diff
@@ -0,0 +1,31 @@
+const deleteWarningText = (element) => {
+  const deleteTarget = element.parentElement.lastElementChild;
+  if (deleteTarget.className === "alert-danger") {
+    deleteTarget.remove();
+  }
+};
+
+const toggleWarningborder = (element) => {
+  if (element.parentElement.lastElementChild.className === "alert-danger") {
+    element.classList.toggle("warning-alert-border");
+  }
+};
+
+const createWarningText = (element, message) => {
+  if (element.parentElement.lastElementChild.className !== "alert-danger") {
+    const warningText = document.createElement("div");
+    warningText.textContent = message;
+    warningText.classList.add("alert-danger");
+    element.parentElement.appendChild(warningText);
+  } else {
+    element.parentElement.lastElementChild.textContent = message;
+  }
```

Comment:
일반적으로 긍정조건이 먼저 나오는게 인지부하가 적다고 알고 있어서 제안드려요.

e.g. 00가 아니면 0하고 아니면(아닌게 아니면) 0한다 => 00하면 0하고 아니면 0한다.
- https://labs.brandi.co.kr/2018/02/13/chunbs.html

그리고, 1줄짜리가 먼저 오는게 더 읽기 쉬울거같아요.

```suggestion
  if (element.parentElement.lastElementChild.className === "alert-danger") {
    element.parentElement.lastElementChild.textContent = message;
  } else {
     const warningText = document.createElement("div");
    warningText.textContent = message;
    warningText.classList.add("alert-danger");
    element.parentElement.appendChild(warningText);
  }
```

Reply:
긍정 조건이 먼저 나오게 반영하겠습니다!